src/effect-store.ts

```diff
@@ -23,8 +23,8 @@ export const {
  * clear the effect store
  * usually called when we want to empty the effect store
  */
-export const clearEffectStore = (effectName: string) => {
-	const effectStore = getEffectStore(effectName);
+export const clearEffectStore = (effectStoreName: string) => {
+	const effectStore = getEffectStore(effectStoreName);
```

Comment:
this was unclear, and threw me off, so renamed this to be explicitly the EffectStore's Name.